libs/blocks/global-navigation/features/breadcrumbs/breadcrumbs.js

```diff
@@ -99,7 +100,9 @@ const fromUrl = () => {
 
 export default async function init(el) {
   try {
-    const breadcrumbsEl = (await createWithBase(el)) || createBreadcrumbs(el) || fromUrl();
+    const breadcrumbsEl = (await createWithBase(el || undefined))
```

Comment:
Small note since this line might seem weird: In the global-navigation.js we init with `init(this.el.querySelector('.breadcrumbs'))` 

An empty querySelector will return null, which in this case will overwrite the default parameter we set for the `createWithBase` - hence we need the "undefined" in the case that `el` is actually null.